### ISSUE TITLE
Package HoloIRC for F-Droid

### ISSUE THREAD

**Issue by rodneyrod** [tilal6991/HoloIRC#65]

**Comment by glitchphan** [comment]
Yeah and we come here to submit bug reports :). I'm mostly using Yaairc because it's in F-Droid and using APKs is a PITA.

**Comment by akien-mga** [comment]
I would also look forward to having HoloIRC on F-Droid.

**Comment by est31** [comment]
Okay, had a look too, seems there are still some issues:

* F-Droid doesn't really work well with manually included libraries like bus or relay, especially as we have to guess which commits of those libraries to build with. Could you use [jitpack](https://jitpack.io) for it instead?

* You include the sonatype repo, which libraries come from it? Are they safe to remove?

Maybe more issues arise when these two are resolved, often you only find out something when actually trying a build.

**Comment by Techman** [comment]
I'm not sure if you guys picked up on this or not, but HoloIRC isn't under active development anymore. It will be replaced in the future anyways, so no use having this discussion.
